app/src/ingestion/scrapy_dst/spiders/edd_spider.py

```diff
@@ -38,6 +38,7 @@ class EddSpider(CrawlSpider):
                     "en/jobs_and_training/Information_Notices/wsin",
                     # Irrelevant pages
                     r"en/.*social-media-toolkit",
+                    r"en/.*Social-Media-Toolkit",
```

Comment:
The must have renamed the webpage

Reply:
Case-insensitive regex 
```suggestion
                    r"(?i)en/.*Social-Media-Toolkit",
```